x/participation/module_simulation.go

```diff
@@ -9,11 +9,16 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/simulation"
 
 	"github.com/tendermint/spn/testutil/sample"
+	participationsim "github.com/tendermint/spn/x/participation/simulation"
 	"github.com/tendermint/spn/x/participation/types"
 )
 
 const (
-// this line is used by starport scaffolding # simapp/module/const
+	opWeightMsgParticipate = "op_weight_msg_create_chain"
```

Comment:
```suggestion
	opWeightMsgParticipate = "op_weight_msg_participate"
```

Reply:
There was an inconsistency in the template https://github.com/tendermint/starport/pull/2154